common/saber_base.h

```diff
@@ -196,12 +200,42 @@ public:                                                         \
     }
   }
 
+#ifdef ENABLE_MENU_DIMBLADE
+  static uint32_t GetCurrentBrightness() {
+    return current_brightness_;//value from 0~16384
+  }
+  // For smooth updates or restore.
+  static void SetBrightness(float v) {
+    current_brightness_ = (int)(v * 16384 / 100.0);  
+    current_brightness_ = clampi32(current_brightness_, 0, 16384) ;
+  }
+
+  enum DimChangeMode {
```

Comment:
There is no need for an enum that only has two values, a "bool" will work just as well.

Reply:
My idea was for future implementation, were you don't want a smooth volume adjustment, but stepwise, in 10 levels for instance. Comparable to ColoChange<>.
Current implementation only allows for smooth color change. By adding DIM_CHANGE_MODE_STEPPED you could make an volume adjustment that steps trough a number of values.